src/components/report/DocumentReport.vue

```diff
@@ -108,15 +127,46 @@
   const onFilterByCategory = (category) => {
     activeCategory.value = category;
   };
+  const onFilter = (item) => {
+    searchText.value = item;
+  };
+  const onClearFilters = () => {
+    activeCategory.value = null;
+    activeSeverity.value = null;
+    if (searchText.value) {
+      searchText.value = null;
+      router.push(route.path);
+    }
+  };
+
+  const hasHiddenCategories = (categories) => categories && categories.some((category) => !category.show);
+  const hasHiddenSeverities = (severities) => {
+    if (severities) {
+      return severities.some((severity) => severity.types.some((type) => !type.show));
+    }
+
+    return true;
+  };
+
+  const hasActiveFilter = () =>
+    hasHiddenCategories(categories.value) || hasHiddenSeverities(severities.value) || !!searchText.value;
 </script>
 
 <template>
   <div class="-mx-3.5 flex flex-wrap">
     <div v-if="hasMessages" class="relative flex shrink grow flex-col sm:w-full md:basis-1/3">
       <div class="sticky top-0 m-2.5">
-        <h3 class="text-xl font-bold">Filters</h3>
+        <h3 class="text-xl font-bold">Search and filter</h3>
         <div class="bg-slate-300">
           <div class="px-4 py-2">
+            <SearchFilter
+              placeholder="Search ..."
```

Comment:
```suggestion
              placeholder="Search title and identifier..."
```

@davidebukali can you make this change and then I'll approve the PR? I tried but I don't have access rights on the devinit fork.

Reply:
Looks good.